app/mailers/reminder_mailer.rb

```diff
@@ -7,6 +7,15 @@ def overdue_email(user, carrier_name, current_location, due_date)
     @due_date = due_date
     mail(to: @user.email, subject: 'Baby Carrier Overdue For Return')
   end
+
+  def week_advance_notice_email(user, carrier_name, current_location, due_date)
+    @user = user
+    @first_name = user.full_name
```

Comment:
We don't need this ivar, we can use `@user` ivar to access `full_name` directly in the mailer template